mica-search/src/main/java/org/obiba/mica/search/queries/NetworkQuery.java

```diff
@@ -172,7 +172,8 @@ public void processHits(QueryResultDto.Builder builder, SearchHits hits, Scope s
   public Map<String, List<String>> getStudyCountsByNetwork() {
     SearchRequestBuilder requestBuilder = searcher.prepareSearch(getSearchIndex()) //
       .setTypes(getSearchType()) //
-      .setSearchType(SearchType.COUNT) //
+      .setSearchType(SearchType.QUERY_THEN_FETCH) //
+      .setSize(0)
       .setQuery(hasQueryBuilder() ? getQueryBuilder() : QueryBuilders.matchAllQuery()) //
       .setNoFields();
```

Comment:
Just a warning for when we do implement ES 5+, some of these setters are no longer available, like _setNofields()_

Reply:
This line is a ES deprecation fix. This code will be moved to plugin, sometime...